kobo/xmlrpc.py

```diff
@@ -479,7 +479,11 @@ def request(self, *args, **kwargs):
                 except KeyboardInterrupt:
                     raise
                 except (socket.error, socket.herror, socket.gaierror, socket.timeout,
-                        httplib.CannotSendRequest, xmlrpclib.ProtocolError) as ex:
+                        httplib.CannotSendRequest, xmlrpclib.ProtocolError,
+                        xmlrpclib.Fault) as ex:
+                    if type(ex) is xmlrpclib.Fault and \
```

Comment:
minor:
More idiomatic and readable would to use `isinstance` instead of `type` and 
`x not in y` instead of `not x in y`.